packages/boilerplate-generator/template.js

```diff
@@ -0,0 +1,12 @@
+// As identified in issue #9149, when an application overrides the default
+// _.template settings using _.templateSettings, those new settings are
+// used anywhere _.template is used, including within the
+// boilerplate-generator. To handle this, _.template settings that have
+// been verified to work are overridden here on each _.template call.
+export default function template(text) {
+  return _.template(text, null, {
```

Comment:
`_` is available globally in the `boilerplate-generator` package since `underscore` is listed as a dependency in `boilerplate-generator/package.js`, but as a matter of practice, should this file specifically declare:

```js
import { _ } from 'meteor/underscore';
```
?

Or even more specifically, just:

```js
import { template as templateProvider } from 'meteor/underscore';
```

?

Reply:
Yes, I was going to add the `import` but noticed the rest of the package was left referencing the global `_`, so I left it as is. I'd prefer to see it added though, so maybe I'll just update all files to have the `import`. Changes coming - thanks!